src/compiler/scala/tools/nsc/reporters/AbstractReporter.scala

```diff
@@ -9,6 +9,8 @@ package reporters
 import scala.collection.mutable
 import scala.tools.nsc.Settings
 import scala.reflect.internal.util.Position
+// TODO
+//import scala.reflect.internal.Reporter
```

Comment:
I actually think those comments are more noise than help; could you remove them across the PR? When we finally get around to removing class `nsc.reporters.Reporter`, it will be simple enough to find the places where we need to import `reflect.internal.Reporter`.